receiver/nsxtreceiver/metadata.yaml

```diff
@@ -0,0 +1,105 @@
+name: nsxtreceiver
+
+resource_attributes:
+  nsx.node.name:
+    description: The name of the NSX Node.
+    type: string
+  nsx.node.id:
+    description: The ID of the NSX Node.
+    type: string
+  nsx.node.type:
+    description: The type of NSX Node.
+    type: string
+    enum:
+      - manager
+      - host
+      - edge
+  nsx.interface.id:
```

Comment:
Should these be `nsxt...`? They can stay as is if we expect they will be exactly the same for a future `nsxv` scraper, but if we are not sure, better to be more specific.

Reply:
Ahh fair point! I will change these 👍